lib/ipsecbase.pm

```diff
@@ -0,0 +1,120 @@
+# SUSE's openQA tests
+#
+# Copyright 2016-2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Summary: Base module for IPsec tests
+# Maintainer: Kernel QE <kernel-qa@suse.de>
```

Comment:
I would add https://www.ipv6ready.org/docs/Phase2_IPsec_Interoperability_Latest.pdf to the comments here on the top (to all 3 modules) and maybe also https://github.com/linux-test-project/ltp/issues/920 to the github commit message.